conda-envs/environment-dev-py37.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
 - mkl-service
 - nbsphinx>=0.4
 - numpy=1.15
-- numpydoc>=0.9
+- numpydoc<1.2
```

Comment:
Would using numpydoc! =1.2.0 allow installation of 1.2.1? The PR fix was merged today, the patch release should hopefully not take long, and with that we could forget about the pin completely

Reply:
While they labeled it `1.2.0` on GitHub, the PyPI release is just `1.2`, so I'm not sure if `!=1.2.0` would work.
Also we have the `<1.2` on `main` already.
But I'm now "watching" the numpydoc repo and will get a notification 🔔 when they do the next release.